package.json

```diff
@@ -44,30 +72,9 @@
     "presets": [
       "@babel/env",
       "@babel/react"
+    ],
+    "plugins": [
+      "@babel/plugin-transform-runtime"
```

Comment:
ooh what's this? Very interested in how everyone has configured jest/babel/webpack etc.